### PR TITLE
Environment Setup

### PR DESCRIPTION
**Pull request by Vidmaster**
Full document for environment setup with Qiskit and Jupyter. Tested and verified as repeatable on OS X, but still requires a Windows user to confirm it works there too.

### CODE REVIEW

**Comment by ZexiXin** [comment]
In the windows, after create the environment, there is no hint like "activate or deactivate the enviroment". But it is not a big deal. And the jupyter-lab cannot run correctly. Hence, we need to use "pip install jupyter" to get jupyter notebook, and use the command "python -m notebook" or "jupyter notebook" the run it.

**Comment by ZexiXin** [comment]
Also, when create python3 notebook, it might give error about the kernel, so we ought to run the "conda install pywin32" to eliminate the kernel error

**Comment by ZexiXin** [comment]
after we have installed the pywin32, someone might also has some problem regarding to the "NoModuleError: no module import cirq", so we need to use "pip install cirq" to make it running.  I guess it is because the pip does work well, so in case the pip.exe doesn't work properly, we need to let the user run pip install for all those packages.

**Comment by ZexiXin** [comment]
figures regarding to Windows Env setting up have been uploaded

File: EnvironmentSetup.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+IBMQ.load_account()
+```
+in a script. At this point you are ready to run a sample circuit in a local simulator and on a quantum computer.
+
+### Sample Circuit Execution
+For reference, the code from the following section is also available in a Jupyter notebook which can be found at [Qiskit-Demo.ipynb](Qiskit-Demo.ipynb).
+
+Create a new Python 3 notebook in Jupyter Lab, and enter the following code from Qiskit's [getting started](https://qiskit.org/documentation/getting_started.html) document in the first cell and execute it with the play button:
+```python
+iimport numpy as np
```
Comment (ZexiXin) [inline EnvironmentSetup.md]: Here should be import , please delete the extra "i" .

**Review by ZexiXin — APPROVED** [review]
looks good to me

**Review by ZexiXin — APPROVED** [review]
looks good to me

File: EnvironmentSetup.md (hunk 1 above)
Comment (Vidmaster) [reply EnvironmentSetup.md]: Good catch, fixed!

**Pull request merged**